frontend/.env.development

```diff
@@ -1,5 +1,6 @@
 REACT_APP_TITLE=SimpleReport
 REACT_APP_BACKEND_URL=http://localhost:8080/graphql
+REACT_APP_PXP_BACKEND_URL=http://localhost:8080/pxp
```

Comment:
This needs to be set in `frontend/package.json`

Alternatively:
I would suggest updating `REACT_APP_BACKEND_URL` to `http://localhost:8080` and then append the `/graphql` and `/pxp` in the app


and set `REACT_APP_BACKEND_URL` to `https://${ENV_NAM}.simplereport.gov/api`

Reply:
I'm going to go with the latter suggestion, it seems cleaner